scripts/go.mod

```diff
@@ -1,138 +1,198 @@
 module _
 
-go 1.17
+go 1.21
```

Comment:
Running the dashboards-linter locally with go 1.21 fails, the way we install dependencies here is probably no longer viable:
```
➜ make dashboards-lint
Installing tools from hack/tools.go
tools.go:9:2: import "github.com/google/go-jsonnet/cmd/jsonnet" is a program, not an importable package
Installing tools from hack/tools.go
tools.go:9:2: import "github.com/google/go-jsonnet/cmd/jsonnet" is a program, not an importable package
Installing tools from hack/tools.go
tools.go:9:2: import "github.com/google/go-jsonnet/cmd/jsonnet" is a program, not an importable package
/home/duologic/git/kubernetes-monitoring/kubernetes-mixin/tmp/bin/jb install
make: /home/duologic/git/kubernetes-monitoring/kubernetes-mixin/tmp/bin/jb: No such file or directory
make: *** [Makefile:22: vendor] Error 127
```